### ISSUE TITLE
You should do intelligent tensor analysis thats multi 1 shot multi step instead of blind time wastage

### ISSUE THREAD

**Issue by lootnath** [openxla/xla#6478]
You should do intelligent tensor analysis thats multi 1 shot multi step instead of blind time wastage

**Comment by cheshire** [comment]
It's a bit hard to follow the issue from the description.

**Comment by penpornk** [comment]
Closing this issue since there hasn't been a follow up in the past 4 months. Please feel free to reopen the issue to provide more clarifications.
